grafana/alertmanager/config.yml

```diff
@@ -0,0 +1,30 @@
+# route:
```

Comment:
Commented code = unused code. Remove or provide further comments why we shouldn't delete this